types/es6-shim/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "es6-shim": "0.34"
```

Comment:
A dependency should only be added if you are including types from this package. es6-shim has no types.